.github/workflows/build_and_test.yaml

```diff
@@ -24,8 +24,12 @@ jobs:
             ROS_REPO: main
           - ROS_DISTRO: rolling
             ROS_REPO: testing
+          - ROS_DISTRO: galactic
+            ROS_REPO: main
+          - ROS_DISTRO: galactic
+            ROS_REPO: testing
     env:
-      UPSTREAM_WORKSPACE: geometric_shapes${{ matrix.env.ROS_DISTRO == 'rolling' && '_rolling' || '' }}.repos
+      UPSTREAM_WORKSPACE: geometric_shapes.repos
```

Comment:
This one should also be removed once `random_numbers` is available for galactic as well